packages/js/core/src/__tests__/Uri.spec.ts

```diff
@@ -50,4 +50,31 @@ describe("Uri", () => {
       path: "uri",
     });
   });
+
+  it("Infers common URI authorities", () => {
```

Comment:
Common URI authorities seem awkward.
How do we decide which authorities are/should be common and which shouldn't?
Can/should different core libraries (js, rust, python) support different common authorities?

This feature would also make it impossible to differentiate between WRAP URIs and other protocol's URIs. (Both would be valid URIs for Uri.parseUri)

Reply:
The "common URI authorities" thing is just the name of the test. It's not part of the feature. I think it's important to test that the feature works with the URI authorities we are commonly using right now. Are there other test cases you have in mind? Would you like more test cases in general?

Regarding the differentiation between WRAP URIs and other protocol URIs with `Uri.parseUri`, I don't think much has changed. Since we don't require the `wrap://` schema prefix, users can already write something like `myAuthority/anyPath.tiktok` and it would be parsed without issue. This feature only changes the accepted URI format such that if an authority is not found but a schema (e.g. `http://`) is present, we assume the schema is the authority of a WRAP URI. It's surface-level formatting flexibility. It doesn't change how URIs interact with the toolchain, or which URIs can resolve to wrappers.